### ISSUE TITLE
what is the validation data for Kumar comparative experiments

### ISSUE THREAD

**Issue by emergencyd** [vqdang/xy_net#4]
what is the strategy for your hyperparameter-tuning? For Kumar comparative experiments, if your network has been adjusted according to Kumar test set, the test result might be unreliable.

**Comment by simongraham** [comment]
Hi @emergencyd - thanks for your comment. When training on Kumar, we used the combined Kumar same tissue and different tissue test sets as a validation set and used the same augmentation strategy for validation patches that we used for the training patches. As shown in the paper, we have shown that the network that has been tuned in this way generalises successfully to other datasets, such as combined CPM and TNBC.
